src/gui/dispersionreticle/translations/translations_zh_cn.json

```diff
@@ -71,54 +71,54 @@
   "hybridReticle.hideStandardReticle.note": [
     "在您仅想使用服务端准星时有用。"
   ],
-  "hybridReticleExtended.label": "Hybrid reticle extended",
-  "hybridReticleExtended.header": "Hybrid reticle extended",
+  "hybridReticleExtended.label": "延时瞄准环外观配置",
+  "hybridReticleExtended.header": "延时瞄准环外观配置",
   "hybridReticleExtended.body": [
-    "Adds configurable reticle displaying current server-side dispersion to client-side reticle.\n",
-    "Other mods does not have influence on the look of this reticle."
+    "指定延时标准环的外观显示效果。\n",
+    "同时使用其他类似模组可能影响显示效果。"
   ],
   "hybridReticleExtended.note": [
-    "Basically, client-side position, but server-side dispersion.\n",
-    "By this, client-side and server-side dispersion desynchronization is clearly visible.\n\n",
-    "Useful if you want to know server-side dispersion, but still want client-side responsiveness."
+    "同时显示客户端瞄准环和服务器实际收到的瞄准环位置。\n",
+    "开启这个选项后客户端瞄准环和服务端实际收到的瞄准环位置差异就清晰可见。\n\n",
+    "在想获知服务端准星位置， 同时又想获得客户端的快速响应速度时较为有用。"
   ],
   "hybridReticleExtended.attention": [
-    "For SPG artillery view, a standard version of this reticle will be used."
+    "火炮视角中此瞄准环将按游戏默认形式显示。"
   ],
   "serverReticle.label": "服务端瞄准环",
   "serverReticle.header": "服务端瞄准环",
   "serverReticle.body": [
     "在客户端准星显示的同时显示服务端瞄准环。\n",
-    "Uses builtin game logic to do so.\n",
+    "使用游戏机制实现。\n",
     "Other mods might influence the look of this reticle.\n\n"
   ],
   "serverReticleExtended.label": "简单服务端瞄准环",
```

Comment:
Simple server reticle is now named server reticle extended

I've renamed it because this reticle is no longer "simple" :)